.taskcluster.yml

```diff
@@ -148,18 +147,18 @@ tasks:
                           - >-
                             cd ..
                             && git clone ${repository}
-                            && cd reference-browser
+                            && cd ${project_name}
                             && git config advice.detachedHead false
                             && git checkout ${head_rev}
-                            && python automation/taskcluster/decision_task.py pull-request
+                            && python automation/taskcluster/decision_task.py raptor --staging
```

Comment:
@JohanLorenzo is this a leftover from testing this patch in staging releases?

Reply:
I suppose this would explain why PR graph [is failing](https://tools.taskcluster.net/task-group-inspector/#/RdXBCuGiQu6Tg_2hso8AfA)

Reply:
I plan to revert the commit that added this line. Although, I do need it to be green first. https://phabricator.services.mozilla.com/D40809 takes care of it.

Reply:
Makes sense, r+'ed there 👍